File: src/main/java/com/brasil/transparente/api/repository/OrgaoRepository.java
package com.brasil.transparente.api.repository;

import com.brasil.transparente.api.entity.Orgao;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import java.util.List;

@Repository
public interface OrgaoRepository extends JpaRepository<Orgao, String> {

    List<Orgao> findByMinisterioMinisterioId(Long ministerioId);

}

File: src/main/kotlin/com/brasil/transparente/api/repository/OrgaoRepository.kt
package com.brasil.transparente.api.repository

import com.brasil.transparente.api.entity.Orgao
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface OrgaoRepository : JpaRepository<Orgao, String> {

    fun findByMinisterioMinisterioId(ministerioId : Long) : List<Orgao>

}